src/main/catalog-sources/kubeconfig-sync.ts

```diff
@@ -156,14 +156,13 @@ export function computeDiff(contents: string, source: RootSource, port: number,
       for (const [contextName, model] of models) {
         // add new clusters to the source
         try {
-          const cluster = new Cluster({ ...model, id: uuid.v4() });
+          const clusterId = createHash("md5").update(`${filePath}:${contextName}`).digest("hex");
```

Comment:
We need stable id for catalog<->cluster relation.

Reply:
Sounds good, I already knew we would need this